packages/cli/project.json

```diff
@@ -2,6 +2,7 @@
   "name": "cli",
   "$schema": "../../node_modules/nx/schemas/project-schema.json",
   "sourceRoot": "packages/cli/src",
+  "bin": "src/cli",
```

Comment:
Does this do anything? I can't find `bin` anywhere in the [`project.json` type definition](https://github.com/nrwl/nx/blob/master/packages/nx/src/config/workspace-json-project-json.ts#L42-L101).

Reply:
I found a description in npm. 

https://docs.npmjs.com/cli/v10/configuring-npm/package-json#bin

You even helped me back then to fix a path problem with it in user-flow CLI :D

Reply:
Yeah, but that's for the `package.json`, not `project.json` :grin:

Reply:
oh my!!! haha.
Good you saw it!

fixed